clar2wasm/src/words/equal.rs

```diff
@@ -0,0 +1,55 @@
+use crate::wasm_generator::{clar2wasm_ty, ArgumentsExt, GeneratorError, WasmGenerator};
+use clarity::vm::{ClarityName, SymbolicExpression, types::TypeSignature, types::SequenceSubtype, types::StringSubtype};
+use walrus::ValType;
+
+use super::Word;
+
+#[derive(Debug)]
+pub struct Equal;
```

Comment:
We don't have a convention yet for it, but shouldn't we call those structs with the same name as the function they deal with? @obycode @krl